AsyncDisplayKit/ASCellNode.mm

```diff
@@ -120,10 +121,10 @@ - (void)setLayerBacked:(BOOL)layerBacked
   ASDisplayNodeAssert(!layerBacked, @"ASCellNode does not support layer-backing.");
 }
 
-- (void)setNeedsLayout
+- (void)__setNeedsLayout
```

Comment:
Actually, I just found this `calculatedSizeDidChange` override point. Let me play with that for a minute and see if we can do better.